publisher/natsPublisher.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
is NatsPublisher used anywhere now? If not, it's still useful, we could put it in a separate pull request.